File: Ejercicios/C#/Clases/Producto.cs
public class Producto
{
    public string Nombre;
    public decimal Precio;

    public Producto()
    {
        Nombre = "Desconocido";
        Precio = 0m;
    }

    public Producto(string nombre, decimal precio)
    {
        Nombre = nombre;
        Precio = precio;
    }

    public Producto(string nombre)
    {
        Nombre = nombre;
        Precio = 10m;
    }

    public Producto(Producto otro)
    {
        Nombre = otro.Nombre;
        Precio = otro.Precio;
    }
}

File: Ejercicios/Java/Clases/Producto.java
public class Producto {
    private String nombre;
    private double precio;

    public Producto() {
        this("Desconocido", 0);
    }

    public Producto(String nombre, double precio) {
        this.nombre = nombre;
        this.precio = precio;
    }

    public Producto(String nombre) {
        this(nombre, 10);
    }

    public Producto(Producto otro) {
        this(otro.nombre, otro.precio);
    }
}
